x/expiration/spec/02_state.md

```diff
@@ -0,0 +1,31 @@
+# State
+
+The expiration module maintains a simple state collection. The `Expiration` is the data structure stored on the blockchain.
+It consists of five parts.
```

Comment:
May want to rephrase and use a colon instead.

`It consists of the following five parts:`

Reply:
fixed.